### DISCUSSION TITLE
Webhook operation lets you pass a body on `GET` requests

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15830] [Feature Requests]
@directus/team  Small(ish) thing, but perhaps we should toggle off the option to pass a body on `GET` requests.

<img width="1455" alt="Screen Shot 2022-09-30 at 9 26 50 AM" src="https://user-images.githubusercontent.com/62306630/193292204-5316fecf-bb72-4507-86f1-c371ef2b7d22.png">

**Comment by azrikahar** [comment]
Not sure whether this was unintentionally left possible for GET requests, but it might also coincide with the (brief) mentions of GET requests with body here: https://docs.directus.io/reference/introduction.html#search-http-method, particularly the fact that Elasticsearch's `search` API (not to be confused with the http SEARCH method) supports GET requests with body. I would personally agree to toggle it off, but also do wonder what are other use cases that may get affected, aside from Elasticsearch that seems to be most often cited around this topic.

There's also a proposed [http `QUERY` method](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-safe-method-w-body-03) so perhaps that should be the recommended method if it officially goes through.

Additional Ref: https://stackoverflow.com/questions/36939748/elasticsearch-get-request-with-request-body

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
